ocpi-tariffs/src/ocpi/v211/tariff.rs

```diff
@@ -41,10 +41,10 @@ pub struct OcpiPriceComponent {
 
     /// Price per unit (excluding VAT) for this tariff dimension
     pub price: Money,
-    /// Minimum amount to be billed. This unit will be billed in this step_size
-    /// blocks. For example: if type is time and step_size is 300, then time will
+    /// Minimum amount to be billed. This unit will be billed in this `step_size`
+    /// blocks. For example: if type is time and `step_size` is 300, then time will
     /// be billed in blocks of 5 minutes, so if 6 minutes is used, 10 minutes (2
-    /// blocks of step_size) will be billed
+    /// blocks of `step_size`) will be billed
```

Comment:
These changes are the result of running `cargo clippy --fix` with `doc_markdown=warn`.